tests/wpunit/PostObjectCursorTest.php

```diff
@@ -16,7 +16,68 @@ public function setUp(): void {
 		$this->admin            = $this->factory()->user->create( [
 			'role' => 'administrator',
 		] );
+		$this->start_count      = 100;
+		$this->start_time       = time();
 		$this->created_post_ids = $this->create_posts();
+
+		register_graphql_fields(
+			'Post',
+			[
+				'testMetaDate' => [
+					'type'    => 'String',
+					'resolve' => function( $source ) {
+						return get_post_meta( $source->ID, 'test_meta_date', true );
+					}
+				],
+				'testMetaNumber' => [
+					'type'    => 'Number',
+					'resolve' => function( $source ) {
+						return get_post_meta( $source->ID, 'test_meta_number', true );
+					}
+				],
+			]
+		);
+
+		add_filter(
```

Comment:
If you `add_filter` in `setUp()`, you need to `remove_filter` in `tearDown()` to prevent side effects in later tests.

Reply:
Either way, I moved it